ceremony/2022-07-12/staged/snapshot.json

```diff
@@ -0,0 +1,56 @@
+{
+	"signatures": [
+		{
+			"keyid": "fc61191ba8a516fe386c7d6c97d918e1d241e1589729add09b122725b8c32451",
+			"sig": "3046022100daec739ea7faf198a2ca94a9a803c5610720cc288f8a958469cde1989c8a119c0221009220aa4b95da4031a5f198a7ec28a6a8fb653ad33e9e47a6c254c488d612fde5"
+		}
+	],
+	"signed": {
+		"_type": "snapshot",
+		"expires": "2022-08-03T14:30:15Z",
+		"meta": {
+			"rekor.json": {
+				"hashes": {
+					"sha256": "9d2e1a5842937d8e0d3e3759170b0ad15c56c5df36afc5cf73583ddd283a463b",
+					"sha512": "176e9e710ddddd1b357a7d7970831bae59763395a0c18976110cbd35b25e5412dc50f356ec421a7a30265670cf7aec9ed84ee944ba700ec2394b9c876645b960"
+				},
+				"length": 797,
+				"version": 3
+			},
+			"revocation.json": {
+				"hashes": {
+					"sha256": "43067322d02cbc715824f2b3e2d38faaa82a79da5db6cb3e03a5128fccc8b091",
+					"sha512": "b81095381ff89550768f2297417c56afa0a4c119e19ddc343388e20fc95ea5a75639b25cfad86db8b4e4002284eddaa6b94455c4ec2a03b070ed84db209cb0e8"
+				},
+				"length": 783,
+				"version": 1
+			},
+			"root.json": {
```

Comment:
found a bug in go-tuf. it's fortunately not breaking and clients don't even query for root metadata hashes in snapshot anymore. but` root.json` USED to be snapshotted in `snapshot.json`.

Taking a look at this code:
https://github.com/theupdateframework/go-tuf/blob/9d0031b6e8ba3115cdea083aad6f1398180659bf/repo.go#L1280-L1287

you can see that old `snapshot.Meta` is not cleared. we only add to the map. once `root.json` was removed from ` r.snapshotMetadata()` (at version 2), we never updated this.

Reply:
@mnm678 @ethan-lowman-dd @joshuagl @znewman01  i will take care of this bug fix right after.